tests/cds-videos/test_videos_transform_rules.py

```diff
@@ -859,3 +859,40 @@ def test_legacy_indico_id_transform(dumpdir, base_app):
         ]
         for identifier in indico_ids:
             assert not identifier.startswith("a")
+
+
+def test_transform_curation(dumpdir, base_app):
+    """Test tags 852, 340, 595 transformed to _curation."""
+    with base_app.app_context():
+        # Load test data
+        data = load_json(dumpdir, "lecture.json")
+
+        # Add `streaming video` to physical medium
+        modified_data = data[1]
+        record_marcxml = modified_data["record"][-1]["marcxml"]
+        modified_data["record"][-1]["marcxml"] = add_tag_to_marcxml(
+            record_marcxml, "340", {"a": "Streaming video"}
+        )
+
+        # Extract record
+        res = load_and_dump_revision(data[1])
+
+        # Assertions
+        assert "_curation" in res
+        curation = res["_curation"]
+        assert "physical_location" in curation
+        assert "physical_medium" in curation
+        assert "internal_note" in curation
+
+        physical_location = curation["physical_location"]
+        for item in physical_location:
+            assert item.startswith("852__")
```

Comment:
Can you be explicit here with the assertion? Assert the exact value we are expecting...